feature-libs/quote/occ/converters/occ-quote-action-normalizer.ts

```diff
@@ -52,13 +59,35 @@ export class OccQuoteActionNormalizer implements Converter<OccQuote, Quote> {
     return { type, isPrimary: primaryActions.includes(type) };
   }
 
-  protected getOrderedActions(state: QuoteState, list: QuoteActionType[]) {
+  protected getOrderedActions(
+    state: QuoteState,
+    list: QuoteActionType[],
+    quoteId: string
+  ) {
     const order = this.quoteConfig.quote?.actions?.actionsOrderByState?.[state];
-
-    return order
-      ? list
-          .filter((item) => order.includes(item))
-          .sort((a, b) => order.indexOf(a) - order.indexOf(b))
-      : list;
+    if (order) {
+      //deep copy order list
+      const clonedActionList = order.map((actionType) => actionType);
```

Comment:
'minor': consider using the new global method `structuredClone` that has been introduced for deep copying.
For more, see: https://developer.mozilla.org/en-US/docs/Web/API/structuredClone